.prow/scripts/test-end-to-end.sh

```diff
@@ -206,7 +206,7 @@ ORIGINAL_DIR=$(pwd)
 cd tests/e2e
 
 set +e
-pytest --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
+pytest basic-ingest-redis-serving.py --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
```

Comment:
Should we be adding back the other tests? I assume this was an oversight.

Reply:
We could perhaps have the top level `.py` file just become an orchestrator for tests in subfolders

Reply:
Other test will be a separate prow job, which i will apply after this PR is merged in

Reply:
Ok